.github/workflows/yamllint.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: yamllint
+on:
+  pull_request:
+    branches:
+    - '*'
+jobs:
+  yamllint:
+    name: yamllint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
why `actions/checkout@v2` and not `actions/checkout@v3` like the other workflows?

Reply:
The default config gives many errors such as the wrong indentation, missing document start "---" and lines too long. So I went through all the rules in the [default config](https://yamllint.readthedocs.io/en/stable/configuration.html#default-configuration) and changed the rules.

Reply:
I missed this update, I'll change it.